platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dsl/dependencies/DefaultDependencyHandler.java

```diff
@@ -392,7 +393,7 @@ public Dependency testFixtures(Object notation) {
         if (testFixturesDependency instanceof ModuleDependency) {
             // Changes here may require changes in DefaultExternalModuleDependencyVariantSpec
             ModuleDependency moduleDependency = (ModuleDependency) testFixturesDependency;
-            moduleDependency.capabilities(c -> c.requireFeature(TEST_FIXTURES_CAPABILITY_FEATURE_NAME));
+            moduleDependency.capabilities(c -> c.requireSuffix(TEST_FIXTURES_CAPABILITY_SUFFIX));
```

Comment:
What about calling `feature` and passing the feature name instead of the manual handling in testfixture and in feature? (This will also add another test for feature implicitly due all the test-fixture tests).